scripts/common-envs.mk

```diff
@@ -14,3 +14,16 @@ ifeq ($(COZYSTACK_VERSION),)
     $(shell git fetch upstream --tags)
     COZYSTACK_VERSION = $(patsubst v%,%,$(shell git describe --tags))
 endif
+
+# Calculate PLATFORM based on current docker daemon arch
+ifndef PLATFORM
+  DOCKER_DAEMON_ARCH := $(shell docker info --format='{{.Architecture}}')
+  ifeq ($(DOCKER_DAEMON_ARCH),x86_64)
+      PLATFORM := linux/amd64
+  else ifeq ($(DOCKER_DAEMON_ARCH),aarch64)
+      PLATFORM := linux/arm64
+  else
+      $(error Unsupported architecture: "$(DOCKER_DAEMON_ARCH)")
+  endif
+  undefine DOCKER_DAEMON_ARCH
+endif
```

Comment:
It would be nice, to break this out into a separate function, like
```
ifndef PLATFORMS
  PLATFORMS := $(get_supported_platforms)
endif
```
since we're eventually going to be pulling up the buildx builder information (not the local docker daemon) and getting the architectures from that.